packages/backend/src/controllers/authentication.ts

```diff
@@ -164,18 +168,27 @@ export const googlePassportStrategy: GoogleStrategy | undefined = !(
               ).href,
               passReqToCallback: true,
           },
-          async (req, issuer, profile, done) => {
+          async (
```

Comment:
This was the main issue with the previous implementation, the callback method was returning an incomplete response , without refreshToken.